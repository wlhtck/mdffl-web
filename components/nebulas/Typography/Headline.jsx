import withStyles from '../withStyles';
import styles from './styles';

const Headline = withStyles(styles({
  color: 'white', family: 'raleway', size: '36px', weight: 800,
}), 'h1');

export default Headline;
