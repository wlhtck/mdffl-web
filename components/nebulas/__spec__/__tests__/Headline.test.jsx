import React from 'react';
import { mount } from 'enzyme';
import 'jest-styled-components';
import Headline from '../../Typography/Headline';

describe('nebulas/Typography/Headline', () => {
  it('renders a <Headine />', () => {
    expect(mount(<Headline>Ready 2 Ball?</Headline>)).toMatchSnapshot();
  });
});
