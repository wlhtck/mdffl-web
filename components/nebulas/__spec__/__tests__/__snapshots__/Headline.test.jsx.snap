// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`nebulas/Typography/Headline renders a <Headine /> 1`] = `
.c0 {
  color: #FFFFFF;
  font-family: 'Raleway',sans-serif;
  font-weight: 800;
  font-size: 36px;
  text-align: left;
  margin: 0 0 32px;
}

<withStyles>
  <StyledComponent
    forwardedComponent={
      Object {
        "$$typeof": Symbol(react.forward_ref),
        "attrs": Array [],
        "componentStyle": ComponentStyle {
          "componentId": "withStyles-sc-8qn4cd-0",
          "isStatic": false,
          "lastClassName": "c0",
          "rules": Array [
            [Function],
          ],
        },
        "displayName": "withStyles",
        "foldedComponentIds": Array [],
        "render": [Function],
        "styledComponentId": "withStyles-sc-8qn4cd-0",
        "target": "h1",
        "toString": [Function],
        "warnTooManyClasses": [Function],
        "withComponent": [Function],
      }
    }
    forwardedRef={null}
  >
    <h1
      className="c0"
    >
      Ready 2 Ball?
    </h1>
  </StyledComponent>
</withStyles>
`;
